docs/pages/versions/unversioned/sdk/device.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Device
+---
+
+Provide information of devices on the application.
+
+## Installation
+
+This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. It is not yet available for [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native apps.
+
+## API
+
+```js
+import * as Device from 'expo-device';
+```
+
+### Constants
+
+- `Device.brand: string`
+
+  Gets the device brand.
+
+  - iOS: `"Apple"`
+  - Android: eg. `"Xiaomi"`
+
+- `Device.carrier: string`
+
+  Gets the carrier name (network operator).
+
+- `Device.manufacturer: string`
+
+  Gets the device manufacturer.
+
+  - iOS: `"Apple"`
+  - Android: eg. `"Google"`
+
+- `Device.model: string`
+
+  **iOS warning**: The list with device names is maintained by the community and could lag new devices. It is recommended to use `deviceId` since it's more reliable and always up-to-date with new iOS devices. We do accept pull requests that add new iOS devices to the list with device names.
```

Comment:
This looks copied straight from react-native-device-info -- write original text.